provider/iot/command.py

```diff
@@ -0,0 +1,79 @@
+from provider.iot.diagram import IoTDiagram
+from shared.command import CommandRunner, BaseCommand
+from shared.common import BaseOptions, ResourceDigest
+from shared.diagram import NoDiagram, BaseDiagram
+
+
+class IotOptions(BaseOptions):
+    thing_name: str
+
+    def __new__(cls, session, region_name, thing_name):
+        """
+        Iot options
+
+        :param session:
+        :param region_name:
+        :param thing_name:
+        """
+        self = super(BaseOptions, cls).__new__(cls, (session, region_name))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Bad first argument 'BaseOptions' given to super()](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)